rfc/tableAvailabilityMarkdown.md

```diff
@@ -0,0 +1,108 @@
+# Table resource supporting filtered availability
+
+This RFC implements [Github Issue 306](https://github.com/yahoo/fili/issues/306)
+
+## Goals
+
+1. Show the queryable range of time as part of the Logical Table endpoint view.
+2. Allow the queryable range to reflect constraints as would be found in a data request.
+  * Required metrics and dimensions
+  * Applied filters
```

Comment:
Do we really need filters?  It seems like the only information they convey are a set of dimensions that are included in the filter request.  The API to return availability seems odd having to specify data filters.  Does the operator matter?  What about the filter values?  Would I get a different result back if I use startsWith vs contains?